### PR TITLE
clean: Remove instructions on how to migrate legacy organizations CY-4234

### PR DESCRIPTION
**Pull request by prcr**
It's no longer possible to follow the instructions to migrate legacy organizations in self-service mode since the settings
page for legacy organizations is inaccessible to users.

For more information follow the [discussion on Slack](https://codacy.slack.com/archives/C02G3V9A3J5/p1633003366018600).

### CODE REVIEW

File: docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Synced organizations have several advantages, such as keeping the repositories a
 
 However, the following scenarios may impact you:
 
--   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. To restore the analysis of your repositories you can wait until Codacy attempts to migrate your organization, or restore the analysis immediately by [performing the migration yourself](../../faq/repositories/how-do-i-migrate-a-legacy-repository-to-a-synced-organization.md).
+-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. To restore the analysis of your repositories please wait until Codacy attempts to migrate your organization.
```
Comment (inesgomas) [inline docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md:22]:
```suggestion
-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories.
```
Sounds it is unnecessary, if feels like an action, but it isn't an action. 

Or we could do something like this:
```suggestion
-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. Codacy will attempt to migrate your organization and restore the analysis of your repositories.
```

**Review by inesgomas — COMMENTED** [review]
Shouldn't we remove this section as well? https://docs.codacy.com/faq/repositories/how-do-i-migrate-a-legacy-repository-to-a-synced-organization/

File: docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md (hunk 1 above)
Comment (prcr) [reply docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md:22]: Yes, I like your second proposal best! Thank you @inesgomas.

**Comment by prcr** [comment]
> Shouldn't we remove this section as well? https://docs.codacy.com/faq/repositories/how-do-i-migrate-a-legacy-repository-to-a-synced-organization/

I don't know if I misunderstood your question, but this pull request deletes that page. :+1:

**Review by inesgomas — APPROVED** [review]

**Pull request merged**